tests/bootloader.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2017
+#
+# All rights reserved.
+#
+# This file is distributed under the Clear BSD license.
+# The full text can be found in LICENSE in the root directory.
+
+import time
+
+import rootfs_boot
+from devices import board, wan, lan, wlan, prompt
+
+class UBootVars(rootfs_boot.RootFSBootTest):
+    '''UBootVars prints U-boot environment variables.'''
+    def runTest(self):
+        board.sendline('fw_printenv && echo \"U-boot environment variables\"')
+        board.expect('boot_partition=(\d)')
```

Comment:
Could we add a couple more random ones just so this test tests for random issues of vars going missing.